### PR TITLE
TC_04.03.009|Verify that the user can open the selected Organization Folder

### PR DESCRIPTION
**Pull request by 7108581**
https://trello.com/c/Z6kTVtkJ/865-at0403009-verify-that-the-user-can-open-the-selected-organization-folder

### CODE REVIEW

**Review by emvrita — APPROVED** [review]

**Comment by DariaMas7** [comment]
Hello, your commit should include AT not TC

File: cypress/e2e/breadcrumbsMyViewsPage.cy.js [hunk 1]
```diff
@@ -81,5 +81,21 @@ describe("breadcrumbsMyViewsPage", () => {
           .should('be.visible')
           .and('include.text', "testMultiBranch Pipeline")
     })
```
Comment (DariaMas7) [inline cypress/e2e/breadcrumbsMyViewsPage.cy.js:83]: There should be empty line between your and the previous it()

File: cypress/e2e/breadcrumbsMyViewsPage.cy.js [hunk 2]
```diff
@@ -81,5 +81,21 @@ describe("breadcrumbsMyViewsPage", () => {
...
+    it('TC_04.03.009|Verify that the user can open the selected Organization Folder', () => {
```
Comment (DariaMas7) [inline cypress/e2e/breadcrumbsMyViewsPage.cy.js:84]: The name is supposed to include AT

File: cypress/e2e/breadcrumbsMyViewsPage.cy.js [hunk 3]
```diff
@@ -81,5 +81,21 @@ describe("breadcrumbsMyViewsPage", () => {
...
+        cy.get('a[href="/view/all/newJob"]').click();
+        cy.get('#name').type('testOrganisation Folder');
```
Comment (DariaMas7) [inline cypress/e2e/breadcrumbsMyViewsPage.cy.js:86]: 'testOrganisation Folder' should be in the fixtures

File: cypress/e2e/breadcrumbsMyViewsPage.cy.js [hunk 4]
```diff
@@ -81,5 +81,21 @@ describe("breadcrumbsMyViewsPage", () => {
...
+        cy.get('.hudson_matrix_MatrixProject').click();
+        cy.get('#ok-button').click();
+        cy.get('.jenkins-button--primary').click();
+
+        cy.get(userDropdown).click({ force: true })
+        cy.get('.yuimenuitem').contains("My Views").click()
+        cy.get('a[class="jenkins-table__link model-link inside"] span')
+          .contains('testOrganisation Folder')
```
Comment (DariaMas7) [inline cypress/e2e/breadcrumbsMyViewsPage.cy.js:94]: the same for this 'testOrganisation Folder' should be in the fixtures

File: cypress/e2e/breadcrumbsMyViewsPage.cy.js [hunk 5]
```diff
@@ -81,5 +81,21 @@ describe("breadcrumbsMyViewsPage", () => {
...
+          .click()
+        cy.get('#main-panel')
+          .should('be.visible')
+          .and('include.text', "testOrganisation Folder")
```
Comment (DariaMas7) [inline cypress/e2e/breadcrumbsMyViewsPage.cy.js:98]: "testOrganisation Folder" -> fixtures

**Review by DariaMas7 — COMMENTED** [review]
lines 85 - 89 might be placed in hook

File: cypress/e2e/breadcrumbsMyViewsPage.cy.js [hunk 6]
```diff
@@ -81,5 +81,21 @@ describe("breadcrumbsMyViewsPage", () => {
...
     })
+    it('TC_04.03.009|Verify that the user can open the selected Organization Folder', () => {
+        cy.get('a[href="/view/all/newJob"]').click();
+        cy.get('#name').type('testOrganisation Folder');
+        cy.get('.hudson_matrix_MatrixProject').click();
+        cy.get('#ok-button').click();
+        cy.get('.jenkins-button--primary').click();
+
+        cy.get(userDropdown).click({ force: true })
+        cy.get('.yuimenuitem').contains("My Views").click()
```
Comment (Rom-T) [inline cypress/e2e/breadcrumbsMyViewsPage.cy.js:92]: Please move `"My Views"` in the fixtures

**Comment by 7108581** [comment]
The new PR # 446 was made. All recommendations and comments were included.

**Pull request closed without merging**